chapters/annotations.tex

```diff
@@ -1117,13 +1117,16 @@ \section{Annotations for the Graphical User Interface}\label{annotations-for-the
 
 This section describes the annotations that are used to define properties of the graphical user interface.
 
-\begin{lstlisting}[language=grammar]
-  preferred-view-annotation:
-     annotation "(" preferredView "=" ("info" | "diagram" | "text") ")"
+\begin{lstlisting}[language=modelica]
+annotation(preferredView = $layer$)
 \end{lstlisting}
 
 The \fmtannotationindex{preferredView} annotation defines the default view when selecting the class.
-\lstinline!info! means info layer, i.e., the documentation of the class, \lstinline!diagram! means diagram layer and \lstinline!text! means the Modelica text layer.
+The $layer$ is a \lstinline!String! literal where \lstinline!"info"! means information layer, i.e., the documentation of the class, \lstinline!"diagram"! means diagram layer, \lstinline!"icon" means icon layer, and \lstinline!"text"! means the Modelica text layer.
```

Comment:
Formatting:
```suggestion
The $\mathit{layer}$ is a \lstinline!String! literal where \lstinline!"info"! means information layer, i.e., the documentation of the class, \lstinline!"diagram"! means diagram layer, \lstinline!"icon" means icon layer, and \lstinline!"text"! means the Modelica text layer.
```

Reply:
What about calling it the _Modelica source code_ instead of _Modelica text layer_?

Reply:
That loses the connection with "text" as layer-name.